package.json

```diff
@@ -82,5 +82,17 @@
   "volta": {
     "node": "20.12.2",
     "pnpm": "9.0.5"
+  },
+  "pnpm": {
+    "patchedDependencies": {
+      "signal-polyfill@0.1.0": "patches/signal-polyfill@0.1.0.patch"
+    }
+  },
+  "dependencies": {
```

Comment:
Should these all be dev dependencies instead?

Reply:
actually -- should all of these move to the react sub-project? and not in the main utils package?
(I just saw that the main package has no changes)